piglow/piglow_test.go

```diff
@@ -0,0 +1,260 @@
+package piglow_test
```

Comment:
package piglow

_test prefix's sole purpose is to make examples copy pasteable without having to fix the namespace.